packages/hash/frontend/src/blocks/page/style.module.css

```diff
@@ -43,6 +43,7 @@
 }
 
 .Block__Content {
+  position: relative;
```

Comment:
Can you add a comment explaining why this is necessary (i.e, for positioning of the placeholder) – this will be really hard to figure out later.

Reply:
Yeah makes sense, added comment line here https://github.com/hashintel/hash/pull/953/commits/4b0726e7562c646b8da0e91c3ccac2c5f2e66c34